File: src/main/java/com/apmosys/employeeportal/dto/QuestionnaireDTO.java
package com.apmosys.employeeportal.dto;

import lombok.*;
import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class QuestionnaireDTO {
    private Long questionId;
    private String questionTitle;
    private String questionDescription;
    private String createdBy;
    private List<QuestionDTO> questions;
    private Long quarterId;
}

File: src/main/frontend/src/app/models/questionnaire-dto.ts

export class QuestionnaireDTO {
  questionId?: number;
  questionTitle: string;
  questionDescription?: string;
  createdBy?: number;
  quarterId?: number;
  quarterCycle?: string;
  departmentId?: number;
  department?: string;
  response: number;
  questions: {
    id?: number;
    questionText: string;
  }[];


}